user/reference/bionic.md

```diff
@@ -107,13 +107,23 @@ For preinstalled language interpreters, a standard version manager like `rvm` is
 
 ## Python support
 
-* Images for all languages have Python 2 and 3 installations provided by the distribution. The Python 2 one has pip preinstalled.
+* Images for all languages have Python 2 (`2.7.17`) and 3 (`3.6.9`) installations provided by the distribution. The Python 2 one is the default and has pip preinstalled.
 * For `language:`: `c`, `cpp`, `node_js`, `elm`, `go`, `generic`, `java`, `groovy`, `clojure`, `php`, `ruby` 
-    * Additionally, extra Python versions are preinstalled, manageable with `pyenv`: `2.7.15`, `3.6.7` and `3.7.1`.
+    * Additionally, extra Python versions are preinstalled, manageable with `pyenv`: `2.7.17`, `3.6.10`, `3.7.6` and `3.8.1`.
 * For `language: python`
-    * The version selected with `python:` is also installed and a `virtualenv` is activated for it. `3.6.7` is the default.
+    * The version selected with `python:` is also installed and a `virtualenv` is activated for it. `3.6.10` is the default.
     * See [Python versions table](/user/languages/python/#python-versions) for a full list of versions available for selection.
 
+| alias  | version  |
+| :----- | :------- |
+| 2.7    | 2.7.17   |
+| 3.6    | 3.6.10    |
```

Comment:
Extra space?

Reply:
Drat! This table shouldn't be there. I'll redo the merge...